packages/account-abstraction/hardhat.config.ts

```diff
@@ -0,0 +1,81 @@
+import '@nomiclabs/hardhat-waffle'
+import '@typechain/hardhat'
+import { HardhatUserConfig } from 'hardhat/config'
+import 'hardhat-deploy'
+import '@nomiclabs/hardhat-etherscan'
+import './tasks/deploy'
+import 'solidity-coverage'
+
+import * as dotenv from 'dotenv'
```

Comment:
On my system this throws an error because the package.json doesn't have "dotenv" as a dependency.

```
diff --git a/packages/account-abstraction/package.json b/packages/account-abstraction/package.json
index aaf9052a9..326486b95 100644
--- a/packages/account-abstraction/package.json
+++ b/packages/account-abstraction/package.json
@@ -79,6 +79,7 @@
     "@thehubbleproject/bls": "^0.5.1",
     "@typechain/hardhat": "^6.1.2",
     "@types/mocha": "^9.0.0",
+    "dotenv": "^16.4.1",
     "ethereumjs-util": "^7.1.0",
     "ethereumjs-wallet": "^1.0.1",
     "hardhat-deploy": "^0.11.12",
```

Reply:
thanks! added